samples/data-grid/cypress/column-cell-events/demo.js

```diff
@@ -0,0 +1,44 @@
+const doc = document;
+
+DataGrid.dataGrid('container', {
+    table: {
+        columns: {
+            product: ['Apples', 'Pears', 'Plums', 'Bananas'],
+            weight: [100, 40, 0.5, 200],
+            price: [1.5, 2.53, 5, 4.5],
+            metaData: ['a', 'b', 'c', 'd']
+        }
+    },
+    defaults: {
+        columns: {
+            editable: true,
+            sorting: true
+        }
+    },
+    columns: {
+        product: {
+            editable: true,
+            events: {
+                afterSorting: function () {
+                    doc.getElementById('columnSorting').value = 'afterSorting';
+                }
```

Comment:
I would move it to the global events options. @stitot maybe you have any suggestions?

Reply:
IMO, we should not mix cell and column events. It's the same situation like series and point.

Reply:
Do you mean to move `column.events` to `events.column`? Hm...

Reply:
It sounds more reasonable and convinced me.

Let's change it.